common/TextDifferencer.java

```diff
@@ -108,7 +115,7 @@ public TextDiff getTextDiff(String leftText, String rightText, String diffString
       if (type == ChangeType.NO_CHANGE) {
         // On NO_CHANGE lines, we know that both diff line indices should be the same. We add
         // placeholder DiffLines to fill in the gaps:
-        fillPlaceholderGap(result, leftDiffLineNumber, rightDiffLineNumber);
+        fillPlaceholderGap(leftLines, rightLines, leftDiffLineNumber, rightDiffLineNumber);
```

Comment:
You need it because you're passing by reference, and you get the changes back.
The "output" of `fillPlaceholderGap` are the changes to`leftLines` and `rightLines`.